Zend/tests/bug79792.phpt

```diff
@@ -3,14 +3,18 @@ Bug #79792: HT iterators not removed if empty array is destroyed
 --FILE--
 <?php
 $a = [42];
-foreach ($a as &$c) {
-    // Make the array empty.
-    unset($a[0]);
-    // Destroy the array.
-    $a = null;
+try {
+    foreach ($a as &$c) {
+        // Make the array empty.
+        unset($a[0]);
+        // Destroy the array.
+        $a = null;
+    }
+} catch (\TypeError $e) {
+    echo $e->getMessage(), \PHP_EOL;
 }
 ?>
 ===DONE===
---EXPECTF--
-Warning: foreach() argument must be of type array|object, null given in %s on line %d
+--EXPECT--
+foreach() argument must be of type array|object, null given
```

Comment:
while there.. Can you please swap the order of types? Because that's the official way ™️